examples/with-stitches/components/StitchesLogo.tsx

```diff
@@ -0,0 +1,48 @@
+export const StitchesLogo = () => (
```

Comment:
Let's do `export default` here.

Reply:
@lfades Done. I couldn't export an anonymous function, the linter didn't like that. I've assigned to a variable and export that as default.